spacy/displacy/__init__.py

```diff
@@ -93,15 +95,20 @@ def serve(
     manual (bool): Don't parse `Doc` and instead expect a dict/list of dicts.
     port (int): Port to serve visualisation.
     host (str): Host to serve visualisation.
+    auto_select_port (bool): Automatically select a port if the specified port is in use.
 
     DOCS: https://spacy.io/api/top-level#displacy.serve
     USAGE: https://spacy.io/usage/visualizers
     """
     from wsgiref import simple_server
```

Comment:
Why do we import `wsgiref` locally?

Reply:
this import was [committed](https://github.com/zrpxx/spaCy/commit/c31792aaec68a67ddaac9ef7ea812345eee0a6bf#diff-ec2e6ce08f100314344ecf042422ffcf98220700235baf283104ace0d79a8068) 6 years ago and kept as is, maybe moving it to the top of the file is a good idea?

Reply:
Ah, sorry - let's ignore that. Thought this was introduced by you.